airbyte-integrations/connectors/source-recharge/README.md

```diff
@@ -21,6 +21,7 @@ development environment of choice. To activate it from the terminal, run:
 ```
 source .venv/bin/activate
 pip install -r requirements.txt
+pip install '.[tests]'
```

Comment:
I think we already have this row in 85 row

Reply:
Removed.